mod_ipstamp/src/mod_ipstamp.erl

```diff
@@ -15,20 +15,20 @@
 %% along with 0 A.D.  If not, see <http://www.gnu.org/licenses/>.
 
 -module(mod_ipstamp).
-
--behaviour(gen_mod).
-
--include("ejabberd.hrl").
 -include("logger.hrl").
 -include("xmpp.hrl").
+-behaviour(gen_mod).
 
 -export([start/2,
          stop/1,
          depends/2,
          mod_opt_type/1,
+         mod_options/1,
          reload/3,
+         mod_doc/0,
```

Comment:
What's your motivation to add mod_doc now? I believe this is neither necessary nor does it add any value.

Reply:
In the latest version, there's a need to defined and the mod_doc function when gen_mod behaviour is enabled or it triggers an error ` Warning: undefined callback function mod_doc/0 (behaviour 'gen_mod')` . I bet it wasn't necessary in the older versions but the newer one requires mod_doc defined when gen_mod is  active. There's also a reference on the office ejabberd site. In our case, we defined mod_doc but not using it. we use the mod_doc to add description or info regarding the module. ` mod_doc() ->
   #{}.`. It can be total ignored but as far as its not defined it will trigger an error when installing the module. The module will be installed but in other to avoid the undefined mod_doc (behaviour 'gen_mod'), theres a need to defined it

Reply:
All right, sounds good. Thanks for the explanation.